chalicelib/lib/slack.py

```diff
@@ -144,13 +147,24 @@ def slack_payload_extractor(req):
         return "failed extracting payload", 200
 
 
-def verify_token(token):
-    if token == os.getenv('slack_token'):
+def verify_token(request, token):
+    #https://api.slack.com/docs/verifying-requests-from-slack
+
+    request_timestamp = request.headers['X-Slack-Request-Timestamp']
+
+    # create basestring out of v0 + timestamp + req_body
+    request_basestring = bytes('v0:' + request_timestamp + ':' + request, 'utf-8')
+    slack_signature = bytes(request.headers['X-Slack-Signature'], 'utf-8')
+    slack_token = token
```

Comment:
Any point of this variable instead of just using `token` below?

Reply:
no - we can use token directly! good point